Apps/WebClient/src/ClientApp/app/app.vue

```diff
@@ -1,196 +1,198 @@
-<style lang="scss">

```

Comment:
whole file changed? Must be line endings...

Reply:
my linter must be doing something weird. i'll look into it